src/clj/clojurians_log/slack_messages.clj

```diff
@@ -104,9 +154,139 @@
 
 (defn message->text
   "Convert Slack markup to plain text."
-  [message usernames]
+  [message usernames channel-names]
```

Comment:
The function signature change causes an exception here:
https://github.com/clojureverse/clojurians-log-app/blob/70555154e393b30a3efb8a4dfe4cb0e2a7006a88/src/clj/clojurians_log/views.clj#L52

This probably just tells we need some tests to access the various routes to make sure it at least it doesn't throw an exception.

Reply:
Btw, #31 exercises all currently known route types. So this kind of issue will be caught once that's been merge. 😄